addons/account/models/account_move.py

```diff
@@ -638,9 +638,8 @@ def default_get(self, fields):
 
         #compute the default credit/debit of the next line in case of a manual entry
         balance = 0
-        for line in self._context['line_ids']:
-            if line[2]:
-                balance += line[2].get('debit', 0) - line[2].get('credit', 0)
+        for line in self.env['account.move'].resolve_2many_commands('line_ids', self._context['line_ids']):
```

Comment:
`resolve_2many_commands` is defined on the model level, not specifically on account.move so you can use `self.resolve_2many_commands`

Reply:
You can pass an extra parameter with the fields you need. `fields=['debit', 'credit']`

Reply:
I'm sorry, you are right. It's the line_ids field of account.move of course.

Reply:
@StefanRijnhart I updated the PR and fixed the CLA to the corporate version in a separate commit.

Reply:
PR is looking good! I think legal wise, Stijn Houben needs to commit the addition to the corporate CLS

Reply:
Well, I discussed it internally with the Dynapps developers and they agreed I should do this: https://github.com/odoo/odoo/pull/25336. ;-)